fix localization issues with default locations

marlokessler opened the pull request:
## What does this PR do?

This PR adds the localization function to location labels in BookingPage, AvailableEventLocations and EventSetupTab. It also replaces the labels for the Attendee Address location type with localization keys and add the keys to the English and German localization table. Finally, it corrects two other missspellings in the German localization table.

Fixes # (issue)
- localization issues with default locations.

**Environment**: Staging(main branch) / Production

## Type of change

<!-- Please delete options that are not relevant. -->

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [x] Test by trying it out

## Checklist
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

Commit:
fix localization issues with default locations

PeerRich commented on apps/web/components/booking/AvailableEventLocations.tsx:
```diff
@@ -23,8 +26,8 @@ export function AvailableEventLocations({ locations }: { locations: Props["event
-            <Tooltip content={locationKeyToString(location)}>
-              <p className="truncate">{locationKeyToString(location)}</p>
+            <Tooltip content={t(locationKeyToString(location) ?? "")}>
+              <p className="truncate">{t(locationKeyToString(location) ?? "")}</p>
```
what does ?? "" do? havent seen that before with t() function

marlokessler replied:
It serves as a fallback, if `locationKeyToString` returns`undefined`.

zomars reviewed (approved):
Thank you for your contribution 🙏

zomars commented:
Seems like we have some conflicts to solve first.

Commit:
Merge remote-tracking branch 'origin/main' into fix/localization-for-attendee-input-on-booking-page

marlokessler commented:
> Seems like we have some conflicts to solve first.

Done👍🏼

PeerRich commented:
why are we merging into [calcom:pr/5569](https://github.com/calcom/cal.com/tree/pr/5569) and not into main?

marlokessler commented:
> why are we merging into [calcom:pr/5569](https://github.com/calcom/cal.com/tree/pr/5569) and not into main?

Ooops, sry. changed the merge branch.

PeerRich commented:
@marlokessler nice thank you, lastly can you update your base branch and remove branch protection so we can update the branch too?


![CleanShot 2022-12-26 at 11 28 04](https://user-images.githubusercontent.com/8019099/209538398-8689fd3a-c200-499c-8301-66ee2cc52469.jpg)

marlokessler committed:
Merge branch 'calcom:main' into fix/localization-for-attendee-input-on-booking-page

marlokessler commented:
> @marlokessler nice thank you, lastly can you update your base branch and remove branch protection so we can update the branch too?
> 
> ![CleanShot 2022-12-26 at 11 28 04](https://user-images.githubusercontent.com/8019099/209538398-8689fd3a-c200-499c-8301-66ee2cc52469.jpg)

I updated the base branch👍🏼 The repo is not covered with any branch protection rules, but I added you to the repo, so that you have write access too

Commit:
Merge remote-tracking branch 'origin' into fix/localization-for-attendee-input-on-booking-page

Commit:
Merge branch 'fix/localization-for-attendee-input-on-booking-page' of https://github.com/Connapptivity/cal.com into fix/localization-for-attendee-input-on-booking-page

marlokessler commented:
> This PR currently has a merge conflict. Please resolve this and then re-add the `automerge` label.

Done
